src/main/java/build/buildfarm/server/BuildFarmServer.java

```diff
@@ -65,6 +69,31 @@ public BuildFarmServer(String session, BuildFarmServerConfig config)
     this(session, ServerBuilder.forPort(config.getPort()), config);
   }
 
+  private Collection<String> getAuthTokens(BuildFarmServerConfig config) throws ConfigurationException {
```

Comment:
I would simplify the logic here to read a file from a known location.
the content of the file is the token. 
I don't think we need to support multiple tokens or lines with comments.
we should keep this as simple as possible since it is temporary anyway.

Reply:
I disagree. Temporary could be a while depending on relative priority of our projects. It wasn't much effort at all to support multiple tokens, and we very likely could have the need (our CI plus 1-2 development customers). With comments, I can document who the tokens belong to in that case in the token file.

Reply:
when we have real customers we will have to do real tokens and not this tokens file..
so for now, I suggest to keep it as simple as possible, for the next few months at least, this is a single token for our internal use.
I don't want to be in a situation where we need to modify/re-deploy if we need to revoke tokens.

Reply:
again, when we have real customers, we can just mount our JWT key here and you will to a JWT check with the provided token against the that private JWT key. no I/IO/network calls required.
Even if this is a long term/refresh token.
so supporting multiple access tokens here is kind of pointless.